R/getRecallByNumber.R

```diff
@@ -0,0 +1,23 @@
+# where does this go?
```

Comment:
👀

Reply:
Where do imports like `library("whatever")` go

Reply:
Section 2.16 of the R Packages book shows how to attach a package which will load it into the namespace: `use_package("<packagename>")`. Sections 6.4/6.5 also has a bit of details on this as well. When you want to call the package, just use `<packagename>::<funcname>(<args>)`

You can also check what packages have already been attached by checking the namespace file.